build.gradle

```diff
@@ -141,7 +141,7 @@ subprojects { subproject ->
 		springSecurityVersion = '5.1.0.BUILD-SNAPSHOT'
 		springSocialTwitterVersion = '1.1.2.RELEASE'
 		springRetryVersion = '1.2.2.RELEASE'
-		springVersion = project.hasProperty('springVersion') ? project.springVersion : '5.0.4.RELEASE'
+		springVersion = project.hasProperty('springVersion') ? project.springVersion : '5.0.5.BUILD-SNAPSHOT'
```

Comment:
What version to use for `4.3.x` back-port ?

Thanks

Reply:
https://github.com/spring-projects/spring-framework/commit/4bc3e0c2b4c703b3da90774e948f8adb18f334f9

4.3.15.BS